src/main/java/org/minions/devfund/noemiguzman/battleship/BattleshipGame.java

```diff
@@ -0,0 +1,83 @@
+package org.minions.devfund.noemiguzman.battleship;
+
+import java.util.Scanner;
+
+/**
+ * Battleship is usually a two-player game, where each player has a fleet and an ocean (hidden from the other player),
+ * and tries to be the first to sink the other player’s fleet.
+ * We will do just a solo version, where the computer places the ships, and the human attempts to sink them.
+ *
+ * We’ll play this game on a 20x20 ocean. This is larger than the ocean in the traditional battleship game.
+ * In this game we will have one 8-square Battleship, one 7-square Battlecruiser, two 6-square Cruisers,
+ * two 5-square Light Cruisers, three 4-square Destroyers and four 3-square Submarines.
+ * Finally, unlike the traditional game, A player can shoot 5 times in each turn.
+ *
+ */
+public class BattleshipGame {
+
+    /**
+     *  In this class you will set up the game; accept 'shots' from the user; display the results;
+     *  print final scores; and ask the user if he wants to play again.
+     *  All input output is done here (although some of it is done by calling a print() method in the Ocean class.)
+     *  All computation will be done in the Ocean class and the various Ship classes.
+     */
+    public BattleshipGame() {
+        System.out.println("Welcome to BattleShip Game! \n");
+    }
+
+    /**
+     * main method.
+     * @param args string
+     */
+    public static void main(final String[] args) {
+
+        BattleshipGame  game = new BattleshipGame();
+        game.payGame();
+    }
+
+    /**
+     * method to play battleship game.
+     */
+
+    private void payGame() {
+        Scanner scanner = new Scanner(System.in, "UTF-8");
+        Ocean ocean = new Ocean();
+
+        ocean.placeAllShipsRandomly();
+
+        while (true) {
+            System.out.println("Shot fired: " + ocean.getShotsFired());
+            System.out.println("Hits: " + ocean.getHitCount());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)